src/ChangeStream.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/*
+ * Copyright 2017 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace MongoDB;
+
+use MongoDB\ChangeStreamIterator;
+use MongoDB\Driver\Manager;
+use MongoDB\Exception\ResumableException;
+use MongoDB\Operation\ChangeStreamCommand;
+
+/**
+ * Operation for the changeStream command.
+ *
+ * @api
+ * @see \MongoDB\Collection::changeStream()
+ * @see http://docs.mongodb.org/manual/reference/command/changeStream/
+ */
+class ChangeStream
```

Comment:
Is there a need for this class to be separate from ChangeStreamIterator?

Since this is the class returned by executing the command (and `Collection::watch()`), I would expect it to implement `Iterator` or `IteratorAggregate`, so that users can [compose](https://en.wikipedia.org/wiki/Object_composition) it with their class expecting an iterable.